R/clean_data.R

```diff
@@ -269,7 +269,5 @@ clean_data <- function(data_joined) {
     )) %>%
     ungroup()
 
-  d$participant_id <- as.integer(factor(d$participant_id))
```

Comment:
Maybe just add this as part of line 50, rather than as an add-on?

Reply:
Wait, sorry this is a deletion. My bad. Why does it need to be in the imp and not the cleaning script?

Reply:
Demog expects it to be a factor and dumps it when it's an integer. So I think makes sense just to have it as an integer when making the imps (which is needed for the multilevel function to work).